### PR TITLE
Fetching api

### PR DESCRIPTION
**Pull request by JoeOsborne77**
App files fetching data from both API's  + formatting and displaying.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,9 @@ cd votewatch
 npm install
 npm start
 ```
+
+
+# Backend install
```
Comment (tomallens) [inline README.md:11]: not sure this is relevant any more!

File: api/divisionsClient.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class DivisionsClient {
```
Comment (tomallens) [inline api/divisionsClient.js:1]: I think this should go too

File: api/test/divisionsClient.test.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+const DivisionsClient = require('../divisionsClient');
```
Comment (tomallens) [inline api/test/divisionsClient.test.js:1]: 😿

**Review by tomallens — APPROVED** [review]
Awesome, this means we meet MVP!

**Review by AUTOMCAS — APPROVED** [review]
Looks good

**Pull request merged**